src/core/secure/keychain.ts

```diff
@@ -4,12 +4,13 @@ import { storeEncrypted, readEncrypted, deleteFromStorage } from './storage';
 import DeviceInfo from 'react-native-device-info';
 import uuidv4 from 'uuid/v4';
 import { Platform } from 'react-native';
+import CONFIG from '../../config/config-beta';
 
 const defaultOptions = {
-    serviceEncryption: 'com.moonlet.encryption',
-    usernameEncryption: 'moonlet-encryption-key',
-    usernamePin: 'moonlet-pin',
-    servicePin: 'com.moonlet.pin'
+    serviceEncryption: `${CONFIG.keychain}.encryption`,
```

Comment:
these are not backword compatible, right ?

Reply:
this is an issue only on iOS ? i remember i saw in plist that both moonlet release and moonlet beta are using sharing the same keystore... maybe it's safer to modify from there. Like we could leave production as it is now and change for beta